lib/components/PublicAuthPrompt.vue

```diff
@@ -70,6 +71,18 @@ watch(() => props.nickname, () => {
 	name.value = props.nickname
 })
 
+watch(() => name.value, (newName) => {
```

Comment:
@susnux I'm not sure here, should it be this instead?

```suggestion
watch(name, (newName) => {
```

Reply:
Only needed for `props` as the prop itself is not reactive so a getter is needed